CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+2. Run the command: `yarn cdk deploy AwsConfigurationStack`.
+
+This deployment will set up:
+
+- Api User with permissions for:
+  - S3
+  - Sns
+  - MediaConvert
+  - MediaLive
+  - Cloudfront
+  - Cloudwatch
+- Media Convert Role with permissions for:
+  - Api Gateway
+  - S3
+- Media Live Role with permissions for:
+  - MediaLive
+  - Cloudwatch
+
+#### Set up the application in local environment
+
+To set up the project locally, follow the instructions provided below. For AWS deployment, refer to the README within the Infrastructure folder.
+
+**Environment Variables:** Add `AWS_MEDIA_CONVERT_ROLE` and `AWS_MEDIA_LIVE_ROLE` with respective ARNs to your Docker Compose file based on your environment (`docker-compose.dev.yml` or `docker-compose.prod.yml`).
+
+#### Create .env file
+
+Create a .env file at the root of the project with all the variables defined in the .env-example file and their respective values.
+
+#### Running the application
+
+Follow these steps to set up and run the application locally:
+
+1. Create a symbolic link to the appropriate Docker Compose file (`docker-compose.dev.yml` or `docker-compose.prod.yml`) for your environment using the following command `ln -s docker-compose.dev.yml docker-compose.yml`.
+2. Run `docker-compose up`.
+3. Run `docker exec -it video-hub bash` to access the video-hub container.
+4. Create a superuser for admin access running `python manage.py createsuperuser`.
+5. Go to `http://localhost:8010/admin` and log in with the superuser credentials.
+
+### How to build the player
+
+In Videoheadline we use the playerReact component to play the videos. This component is built using React and it is compiled to a CSS and JS file that are used in the Django template that renders the player.
+If you need to make changes to the player, you need to follow the steps below to build the player and test it in Videoheadline.
+
+#### Steps to create a new version of the Videoheadline playerReact:
+
+1. Make the changes and compile using npm run build.
+2. Copy the compiled CSS and JS to player/static/player/css and player/static/player/js, respectively.
+3. Delete the old CSS and JS files in player/static/player/css and player/static/player/js.
+4. In the player/templates/player/index.html template, change the names of the CSS and JS files to which the template points.
+5. Test in Videoheadline to ensure that the player is working correctly.
+
+### How to run the tests
+
+To run the tests, follow the steps below:
+
+1. Once the docker is running, run the command: `docker exec -it video-hub bash` to access the video-hub container.
+2. Run the command: `python manage.py test` to run the tests.
+
+### How to use the linters
+
+If you want to contribute to the project, it is important to use the linters to ensure that the code is consistent and follows the best practices. The linter used in the project is ESLint for JavaScript. In addition, we use Prettier to format the JavaScript code.
```

Comment:
Lets add a note that says something like "we plan to also include a Python linter in the future"